project-base/storefront/components/Forms/TextInput/SearchInput.tsx

```diff
@@ -63,7 +63,7 @@ export const SearchInput: FC<SearchInputProps> = ({
                 type="submit"
                 onClick={onSearch}
             >
-                <SearchIcon className="w-[18px]" />
+                <SearchIcon className="w-[18px] text-inputPlaceholder" />
```

Comment:
To make it unified, I think we can remove `w-[18px]` and use `size-4` or `size-5`.